controller/controller.go

```diff
@@ -184,23 +200,28 @@ func (c *controller) updateIngresses() (err error) {
 
 					serviceName := serviceName{namespace: ingress.Namespace, name: path.Backend.ServiceName}
 
-					if address := serviceMap[serviceName]; address != "" {
+					if address := serviceMap[serviceName]; address == "" {
+						skipped = append(skipped, fmt.Sprintf("%s/%s (service doesn't exist)", ingress.Namespace, ingress.Name))
+					} else if !c.ingressNameSupported(ingress) {
+						skipped = append(skipped, fmt.Sprintf("%s/%s (unsupported ingress name [%s]; this feed supports only [%s])",
+							ingress.Namespace, ingress.Name, ingress.Annotations[ingressNameAnnotation], c.ingressName))
+					} else {
 						entry := IngressEntry{
-							Namespace:             ingress.Namespace,
-							Name:                  ingress.Name,
-							Host:                  rule.Host,
-							Path:                  path.Path,
-							ServiceAddress:        address,
-							ServicePort:           int32(path.Backend.ServicePort.IntValue()),
-							Allow:                 c.defaultAllow,
-							StripPaths:            c.defaultStripPath,
-							ExactPath:             c.defaultExactPath,
-							BackendTimeoutSeconds: c.defaultBackendTimeout,
+							Namespace:      ingress.Namespace,
+							Name:           ingress.Name,
+							Host:           rule.Host,
+							Path:           path.Path,
+							ServiceAddress: address,
+							ServicePort:    int32(path.Backend.ServicePort.IntValue()),
+							Allow:          c.defaultAllow,
+							StripPaths:     c.defaultStripPath,
+							ExactPath:      c.defaultExactPath, BackendTimeoutSeconds: c.defaultBackendTimeout,
 							BackendMaxConnections: c.defaultBackendMaxConnections,
 							ProxyBufferSize:       c.defaultProxyBufferSize,
 							ProxyBufferBlocks:     c.defaultProxyBufferBlocks,
 							CreationTimestamp:     ingress.CreationTimestamp.Time,
 							Ingress:               ingress,
+							IngressName:           ingress.Annotations[ingressNameAnnotation],
```

Comment:
this is also super confusing. 
```suggestion
							IngressControllerName:           ingress.Annotations[ingressControllerNameAnnotation],
```